.github/workflows/actions.yml

```diff
@@ -793,3 +793,42 @@ jobs:
         with:
           name: binaries-${{ github.job }}
           path: pfx/**/*
+
+  android:
+    needs: [ what-to-make ]
```

Comment:
This depends on `what-to-make` but builds regardless of its findings. Please extend `what-to-make` with a `make-android` output (using `get_changes core CMakeLists.txt cmake third-party libtransmission android`) and test for it here to avoid needless builds.